debugger/lldbPythonExtension/lldb_sgx_plugin.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from __future__ import print_function
+import struct
+import os.path
+import os
+import lldb
+from ctypes import create_string_buffer
+import load_symbol_cmd
+import ctypes, mmap, errno
+from threading import Thread, Lock
+
+
+POINTER_SIZE = 8
+
+# These constant definitions must align with _oe_enclave structure defined in host\enclave.h
+OE_ENCLAVE_MAGIC_FIELD = 0
+OE_ENCLAVE_ADDR_FIELD = 2
+OE_ENCLAVE_HEADER_LENGTH = 0X28
+OE_ENCLAVE_HEADER_FORMAT = 'QQQQQ'
+OE_ENCLAVE_MAGIC_VALUE = 0x20dc98463a5ad8b8
+
+OE_ENCLAVE_FLAGS_OFFSET = 0x588
+OE_ENCLAVE_FLAGS_LENGTH = 2
+OE_ENCLAVE_FLAGS_FORMAT = 'BB'
+OE_ENCLAVE_THREAD_DATA_OFFSET = 0x28
+
+# These constant definitions must align with ThreadData structure defined in host\enclave.h
+THREAD_DATA_SIZE = 0x28
```

Comment:
Again all references to THREAD_DATA were modified to THREAD_BINDING to match the current data structure. My previous comment is applicable here too. But please synch your file with the master gdb_sgx_plugin.py.